corgi-concourse-resource/corgi_concourse_resource/utils.py

```diff
@@ -2,6 +2,20 @@
 import sys
 
 
+def get_collection_id(job):
+    repo = job["repository"]
+    book_slug = job["books"][0]["slug"]
+    owner = repo["owner"]
+    repo_name = repo["name"]
+    if owner != "openstax":  # pragma: no cover
+        return f"{owner}/{repo_name}/{book_slug}"
+    return f"{repo_name}/{book_slug}"
+
+
+def get_style(job):
+    return job["books"][0]["style"]
```

Comment:
(not important, just for discussion) With the new CORGI isn't this always `default`? I thought we decided that if folks want to test with different recipes they can create a branch, no?

Reply:
Yep, this is only used to pass the style to the Enki pipeline. I guess the pipeline parses it from the books.xml too, so we can probably do away with this property in the input.

You can see an example of what is fetched at `corgi-pr-514-enki/jobs/build-pdf/builds/169` in concourse.